client/Windows/wf_graphics.c

```diff
@@ -237,24 +237,35 @@ static BOOL wf_Pointer_New(rdpContext* context, const rdpPointer* pointer)
 		            pointer->height);
 		info.hbmMask = CreateBitmap(pointer->width, pointer->height, 1, 1, pdata);
 		_aligned_free(pdata);
-		pdata = (BYTE*) _aligned_malloc(pointer->width * pointer->height *
-		                                GetBitsPerPixel(gdi->dstFormat), 16);
 
-		if (!pdata)
+		UINT32 srcFormat;
```

Comment:
Declarations at beginning of a scope only, older compilers (hello VS2010) are not really C99 conformant...

Reply:
got it